### ISSUE TITLE
Crashes on Android when swiping down and no data

### ISSUE THREAD

**Issue by ryan-cat** [rgommezz/react-native-scroll-bottom-sheet#5]
First off, great work! Really needed something that had this functionality.

### Current Behavior

I was using the example from the ReadMe. Only on android, when the data provided to the FlatList is an empty array, swiping up to open the bottom sheet fully, works perfectly. However, when swiping down to close the bottom sheet, the app crashes with the following error: "Invariant Violation: scrollToIndex out of range: requested index 0 but maximum is -1". This error does not occur ever on iOS and does not occur when there is data in the data property to the FlatList on Android.

![Screen Shot 2020-05-21 at 11 11 51 AM](https://user-images.githubusercontent.com/14203213/82573622-0021e180-9b54-11ea-87ee-25f9cf665b5a.png)

### Expected Behavior

I would expect for the the bottom sheet to snap to either the middle snap point or the bottom snap point.

### How to reproduce

Starting from the example from the ReadMe, change the length of the data array to be 0. 
Swipe up to fully open the bottom sheet on Android. Then, swipe down to close the bottom sheet, and the app should crash.

```
<ScrollBottomSheet
      ref={bottomSheetRef}
      componentType="FlatList"
      initialSnapIndex={2}
      snapPoints={['4%', '55%', '90%']}
      data={Array.from({ length: 0 }).map((_, i) => String(i))}
      keyExtractor={(i) => i}
      renderHandle={() => (
        <View style={styles.header}>
          <View style={styles.panelHandle} />
        </View>
      )}
      renderItem={({ item }) => (
        <View style={styles.item}>
          <Text>{`Item ${item}`}</Text>
        </View>
      )}
      contentContainerStyle={styles.contentContainerStyle}
    />
```

### Your Environment

| package                           | version
| ----------------                  | -------
| react-native-scroll-bottom-sheet  | 0.6.0
| react-native                      | 0.61.5
| react-native-gesture-handler      | 1.6.0
| react-native-reanimated           | 1.8.0

**Comment by rgommezz** [comment]
Hello @ryan-cat, thanks for the detailed report, that helps :)

I've market it as a bug, will address it asap.

**Comment by raghavkulkarni1221** [comment]
Is it fixed? Because i am experiencing the exact same problem. If its not fixed, is there some workaround for now atleast??

**Comment by ryan-cat** [comment]
@rgommezz is there any update on this?

**Comment by rgommezz** [comment]
Fixed on `0.6.2`
